### DISCUSSION TITLE
Add ability for community admins to ban "ghost"/spam users from a given community

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#2287] [Ideas]
Over at [COPIM](https://copim.pubpub.org/), we are experiencing infrequent but increasing additions of Pubs by users that are not members of our community (hence 👻 users).

While I understand those users won't be able to publish Pubs w/o going through a formal `Request Publication` process, I was wondering if there might be a way to implement an option to fully ban these users from even starting up draft Pubs inside our community?

I'm not sure if other communities also experience this kind of spamming behaviour by unknown users, so this might just be a fringe case ... it's just that we've had these a couple of times in the past year, and now with three quite recent entries on 

* "What Are the Best Online Movie Rental Sites" [link1](https://copim.pubpub.org/pub/moy7lsng/draft) 
* "The days of having to make a trip to the video store" [link2](https://copim.pubpub.org/pub/ocjg6v0c/draft)
* "5 Red Flags to Look Out for Before Getting Your Essay Written Online In 2022" [link3](https://copim.pubpub.org/dash/pub/usnhps0g/draft)

(all of which are spam entries completely unrelated to what our community is working on 😉). 

For now, I can of course just go ahead and delete these Pub drafts, so just a thought ...

**Comment by tosteiner** [comment]
Might also relate to these other issues/discussion points [#1382](https://github.com/pubpub/pubpub/discussions/1382) and #1498

**Comment by gabestein** [comment]
Hi Tobias - there’s a community-level setting called “Public New Pub Button” that allows for anyone to create a new pub (but not publish it) that I believe you have on that’s mostly intended for conferences and open communities. Turning it off should prevent this - and we’re also working on more holistic ways to prevent and remove spam across PubPub.

**Reply by tosteiner** [reply]
oooh, I see - perfect, that should help, thanks so much, Gabe!
